### PR TITLE
Github assignees are now reviewers

### LINKED ISSUE #410: Update 'reviewers' to use GitHub reviewers

### PR DESCRIPTION
**Pull request by eanker**
Fixes #410

---
Review this pull request [on Preview Code](https://preview-code.com/projects/preview-code/frontend/pulls/430/overview).

### CODE REVIEW

File: src/projects/project-activity.html [hunk 1]
```diff
@@ -30,8 +30,6 @@
 <link rel="import" href="../../bower_components/iron-icons/iron-icons.html">
 
 <link rel="import" href="../ajax/github-authenticated-ajax.html">
-<link rel="import" href="../list-items/list-item-label.html">
-<link rel="import" href="../list-items/column-list-item.html">
```
Comment (txsmith) [inline src/projects/project-activity.html:34]: Were these imports unused?

File: src/projects/pull-request/pull-request-general-information.html [hunk 2]
```diff
@@ -105,7 +105,7 @@
         <div class="sidebar">
           <pr-status id="PRStatus" pull-request="[[pullRequest]]" status="{{_prStatus}}"></pr-status>
           <pull-request-reviewers
-              editable="[[editable]]" pull-request="[[pullRequest]]"
+              editable="[[editable]]" pull-request="{{pullRequest}}"
```
Comment (txsmith) [inline src/projects/pull-request/pull-request-general-information.html]: Why was this change needed?

File: src/projects/pull-request/set-reviewers.html [hunk 3]
```diff
@@ -55,7 +55,19 @@
         reviewer: {
           type: String,
           notify: true
+        },
+        reviewers: {
+          type: Array,
+          computed: '_getReviewers(project.assignees, owner)'
```
Comment (txsmith) [inline src/projects/pull-request/set-reviewers.html:61]: This still refers to `assignees`. Is that correct?

File: src/projects/pull-request/set-reviewers.html [hunk 4]
```diff
@@ -55,7 +55,19 @@
...
+        },
+        owner: {
+          type: String
         }
+      },
+      _getReviewers: function(assignees, owner) {
+        return assignees.filter(function(assignee) {
+          return assignee.login !== owner;
+        });
```
Comment (txsmith) [inline src/projects/pull-request/set-reviewers.html:70]: If the above needs a fix, please rename it here too

File: src/projects/project-activity.html (hunk 1 above)
Comment (eanker) [reply src/projects/project-activity.html:34]: Yes, they both are not used in that document.
Shouldn't have committed it with these changes though.

File: src/projects/pull-request/set-reviewers.html (hunk 3 above)
Comment (eanker) [reply src/projects/pull-request/set-reviewers.html:61]: Yes, we fetch the possible reviewers here, and there is no separate list for that only the list with possible assignees.

**Comment by txsmith** [comment]
Works perfectly!
